doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,218 @@
+We then make use of a ``box refinement`` to refine the mesh around the particle and properly evaluate the velocity field with elements of size :math:`1/4 D \times 1/4 D \times 1/4 D`.
+
+.. code-block:: text 
+
+    subsection mesh
+        set type                   = dealii
+            set grid type          = subdivided_hyper_rectangle
+            set grid arguments     = 2,1,1 : -24,-16,-16 : 40,16,16 : true
+            set initial refinement = 4
+        end
+
+    subsection box refinement 
+        set initial refinement = 3
+        subsection mesh
+            set type               = dealii
+            set grid type          = subdivided_hyper_rectangle
+            set grid arguments     = 2,1,1: -6,-4,-4 : 10,4,4 : true
+            set initial refinement = 0
+        end
+    end
+
+Immersed Boundary Particles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We define the particle in flow using the sharp immersed boundary method. The particle is defined as a cylinder with a radius of :math:`D/2 = 0.5` and a half-length of :math:`L/2 = 1.0`. The particle is centered at :math:`(0,0,0)` 
+and the cylider's axis is oriented perpendicularly to direction of the flow. 
+
+The ``local mesh refinement`` is set to 3 to ensure we resolve the boundary layer flow around the particle with elements of size :math:`1/32 D \times 1/32 D \times 1/32 D`.
+
+.. code-block:: text
+
+    subsection particles
+        set number of particles                     = 1
+        set assemble Navier-Stokes inside particles = false
+        subsection extrapolation function
+            set stencil order = 2
+            set length ratio  = 1
+        end
+        subsection local mesh refinement
+            set initial refinement                = 3
+            set refine mesh inside radius factor  = 0.75
+            set refine mesh outside radius factor = 2
+        end
+        subsection particle info 0
+            set type            = cylinder
+            set shape arguments = 0.5;1.0
+            set integrate motion = false
+            subsection position
+                set Function expression = 0;0;0
+            end
+            subsection orientation
+                set Function expression = 1.5707963267948966;0;0
+            end
+        end
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+We define the boundary conditions similarly to the :doc:`../../incompressible-flow/3d-flow-around-sphere/flow-around-sphere` example although, in our case, we won't need a boundary condition defined on the particle as the sharp immersed boundary method will take care of it.
+
+The domain makes use of an inlet velocity of :math:`1~m/s` on the left, slip boundary conditions parallel to the flow direction, and and an outlet with :math:`p = 0` on the right of the domain. In our case, the outlet boundary condition is the same as the natural boundary condition can therefore be left unchanged as Lethe will apply it by default.
```

Comment:
There is a word missing in the last sentence. It can even be a period because the sentence is already long